filebeat/config/config.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Defaults for config variables which are not set
 const (
-	DefaultRegistryFile                      = ".filebeat"
+	DefaultRegistryFile                      = "registry"
```

Comment:
I don't have a strong opinion here. So leaving it as `registry` is fine with me. (I never meant to +1 my own comment anyways; I didn't realize clicking the thumb icon added a +1 until after I clicked it, and then I couldn't figure out how to undo.)

> The reason I used registry is that on DEB/RPM installations the file name already used is /usr/share/filebeat/registry. By using registry we can keep BC for those cases at least.

Those installations were using a non-default value so they should not be affected by the change.

> In addition, JSON is an implementation detail which we might change in the future.

True, leaving it without an extension makes it easier to change underlying file type.